data/projects.json

```
@@ -178,5 +178,15 @@
         "level": "intermediate",
         "site": "",
         "tags": "react, typescript, redux-toolkit"
+    },
+    {
+        "name": "vision-ai",
+        "display": "Vision",
+        "description": "Extract text from an image using a trained AI model",
+        "author": "Angelvicks",
+        "category": "website",
+        "level": "intermediate",
+        "site": "https://github.com/Angelvicks/vision-ai",
+        "tags": "fastapi, react, artificial intelligence"
```

Hi @Angelvicks,
The tag is separate by `-`
```suggestion
        "tags": "fast-api, react, artificial-intelligence"
```

---

The changes are done, thanks for reviewing my pull request.